javascript/javascript1/week3/week3.js

```diff
@@ -0,0 +1,170 @@
+//Item array removal
+let names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+console.log(names);
+
+//When will we be there
+
+function getHours(speed, destinationDistance) {
+  const hours = destinationDistance / speed;
+  function formatTime(hours) {
+    const wholeHours = Math.floor(hours);
+    const minutes = Math.round((hours - wholeHours) * 60);
+    console.log(wholeHours + " hours and " + minutes + " minutes");
+  }
+  formatTime(hours);
```

Comment:
So when you call `getHours` in reality you get `formatTime` in this case, I'd suggest that you have only one function, or split `formatTime` outside of `getHours` but make it take an argument which would be `hours` returned from `getHours`. Hope it makes sense.